spec/system/reward_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Reward test', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+
+  let!(:employee) { create(:employee) }
+  let!(:reward) { create(:reward) }
+
+  it 'test reward' do
+    visit root_path
```

Comment:
Consider using employee login helper here:
https://sajadtorkamani.com/how-to-login-in-capybara-tests-when-using-devise/